site/source/tutorials/1010_containers_and_injecting_state/index.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: false
+title: State management
+overview: In this tutorial, you will learn what a Containers and Injectors are and how to use them to manage external state and inject this into the parts of the widget tree.
+---
+
+{% section 'first' %}
+
+# State management
+
+## Overview
```

Comment:
Given the large number of concepts in this tutorial, it might make sense to consider some sort of "refactoring checklist" that we're going to follow here, with very clear goals in bullet point form at the start. I say this because there's a lot of new terminology and a fair number of new concepts here, and it's difficult as you're ready to know where we're headed. Or maybe I'm just thinking this because of the review process, but it's something to consider at least.